File: dataset/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Task126.java

import java.util.Scanner;
import java.util.HashMap;
import java.util.Map;

public class Task126 {
    private static Map<String, String> pageRoutes = new HashMap<>();
    
    static {
        pageRoutes.put("home", "https://example.com/home");
        pageRoutes.put("about", "https://example.com/about");
        pageRoutes.put("contact", "https://example.com/contact");
        pageRoutes.put("products", "https://example.com/products");
        pageRoutes.put("services", "https://example.com/services");
    }
    
    public static String redirectUser(String userInput) {
        if (userInput == null || userInput.trim().isEmpty()) {
            return "Error: Invalid input";
        }
        
        String normalizedInput = userInput.toLowerCase().trim();
        
        if (pageRoutes.containsKey(normalizedInput)) {
            return "Redirecting to: " + pageRoutes.get(normalizedInput);
        } else {
            return "Error: Page not found. Redirecting to: " + pageRoutes.get("home");
        }
    }
    
    public static void main(String[] args) {
        System.out.println("=== Page Redirect System - Test Cases ===\\n");
        
        // Test Case 1: Valid page - home
        String test1 = "home";
        System.out.println("Test 1 - Input: '" + test1 + "'");
        System.out.println("Result: " + redirectUser(test1));
        System.out.println();
        
        // Test Case 2: Valid page - about
        String test2 = "about";
        System.out.println("Test 2 - Input: '" + test2 + "'");
        System.out.println("Result: " + redirectUser(test2));
        System.out.println();
        
        // Test Case 3: Valid page with mixed case
        String test3 = "PRODUCTS";
        System.out.println("Test 3 - Input: '" + test3 + "'");
        System.out.println("Result: " + redirectUser(test3));
        System.out.println();
        
        // Test Case 4: Invalid page
        String test4 = "unknown";
        System.out.println("Test 4 - Input: '" + test4 + "'");
        System.out.println("Result: " + redirectUser(test4));
        System.out.println();
        
        // Test Case 5: Empty input
        String test5 = "";
        System.out.println("Test 5 - Input: '" + test5 + "'");
        System.out.println("Result: " + redirectUser(test5));
        System.out.println();
    }
}

File: dataset/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Task126.py

def redirect_user(user_input):
    page_routes = {
        "home": "https://example.com/home",
        "about": "https://example.com/about",
        "contact": "https://example.com/contact",
        "products": "https://example.com/products",
        "services": "https://example.com/services"
    }
    
    if not user_input or not user_input.strip():
        return "Error: Invalid input"
    
    normalized_input = user_input.lower().strip()
    
    if normalized_input in page_routes:
        return f"Redirecting to: {page_routes[normalized_input]}"
    else:
        return f"Error: Page not found. Redirecting to: {page_routes['home']}"


if __name__ == "__main__":
    print("=== Page Redirect System - Test Cases ===\\n")
    
    # Test Case 1: Valid page - home
    test1 = "home"
    print(f"Test 1 - Input: '{test1}'")
    print(f"Result: {redirect_user(test1)}")
    print()
    
    # Test Case 2: Valid page - about
    test2 = "about"
    print(f"Test 2 - Input: '{test2}'")
    print(f"Result: {redirect_user(test2)}")
    print()
    
    # Test Case 3: Valid page with mixed case
    test3 = "PRODUCTS"
    print(f"Test 3 - Input: '{test3}'")
    print(f"Result: {redirect_user(test3)}")
    print()
    
    # Test Case 4: Invalid page
    test4 = "unknown"
    print(f"Test 4 - Input: '{test4}'")
    print(f"Result: {redirect_user(test4)}")
    print()
    
    # Test Case 5: Empty input
    test5 = ""
    print(f"Test 5 - Input: '{test5}'")
    print(f"Result: {redirect_user(test5)}")
    print()
